cabal-lint/cabal-lint.cabal

```diff
@@ -0,0 +1,64 @@
+cabal-version:       3.0
```

Comment:
Question for the reviewers: how much of the wheel did we just reinvent? Does a `cabal` linter already exist that we could enrich with any of our desiderata instead of implementing our own thusly? Or is this thing small enough as-is that, regardless of other projects, we'd like to have one tool narrowly-focused on our concerns?